### PR TITLE
Design Forest Fire Popup

### PR DESCRIPTION
**Pull request by prachigarg19**
Component:
image

### CODE REVIEW

**Review by gangakhushlani — APPROVED** [review]

**Comment by mohitb35** [comment]
The background for most of the popup is currently transparent.

Screenshot 2024-04-17 at 12 18 32 PM

File: src/temp/FirePopup/index.tsx [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+              <span>High</span> alert confidence
+            </p>
+            <div className={styles.setUpAlertsContainer}>
+              <p className={styles.setUpAlerts}>
+                Set up alerts with <span>FireAlert</span>
+              </p>
+              <RightArrowIcon width={5} height={8} color={'#4F4F4F'} />
+            </div>
+          </div>
+        </div>
```
Comment (mohitb35) [inline src/temp/FirePopup/index.tsx:71]: How could you structure the html for `popupContainer` for better semantics?

For example, would a `aside` tag be better than using a `div` for the outermost element? How can you identify the heading/label of this element?

File: src/temp/FirePopup/index.tsx
Comment (mohitb35) [inline src/temp/FirePopup/index.tsx:1]: Add a comment to indicate that the contents are hardcoded. This component will need to be refactored based on the api/data that is needed.

**Review by mohitb35 — CHANGES_REQUESTED** [review]
@prachigarg19 Please go through the comments above.

**Review by mohitb35 — APPROVED** [review]

**Pull request merged**